docs/tutorial.rst

```diff
@@ -524,22 +714,28 @@ a) Selecting a range of Zenith Angles
 ****************
 ::
 
-  from pyuvdata import UVBeam
-  import numpy as np
-  import matplotlib.pyplot as plt
-  beam = UVBeam()
-  filenames = ['pyuvdata/data/HERA_NicCST_150MHz.txt', 'pyuvdata/data/HERA_NicCST_123MHz.txt']
-  # have to specify the telescope_name, feed_name, feed_version, model_name
-  # and model_version because they are not included in the file
-  beam.read_cst_beam(filenames, telescope_name='HERA', feed_name='PAPER_dipole',
-                       feed_version='0.1', model_name='E-field pattern - Rigging height 4.9m',
-                       model_version='1.0')
-
-  new_beam = beam.select(axis2_inds=np.arange(0, 20), inplace=False)
-  # plot zenith angle cut through beams
-  plt.plot(beam.axis2_array, beam.data_array[0, 0, 0, 0, :, 0],
-           new_beam.axis2_array, new_beam.data_array[0, 0, 0, 0, :, 0], 'r')
-  plt.xscale('log')
-  plt.xlabel('Zenith Angle (deg)')
-  plt.ylabel('Power')
-  plt.show()
+  >>> from pyuvdata import UVBeam
+  >>> import numpy as np
+  >>> beam = UVBeam()
+  >>> filenames = ['pyuvdata/data/HERA_NicCST_150MHz.txt', 'pyuvdata/data/HERA_NicCST_123MHz.txt']
+  >>> beam.read_cst_beam(filenames, beam_type='power', telescope_name='HERA',
+  ...                    feed_name='PAPER_dipole', feed_version='0.1',
+  ...                    model_name='E-field pattern - Rigging height 4.9m',
+  ...                    model_version='1.0')
+  >>> new_beam = beam.select(axis2_inds=np.arange(0, 20), inplace=False)
+
+  plot zenith angle cut through beams
+  >>> plt.plot(beam.axis2_array, beam.data_array[0, 0, 0, 0, :, 0], # doctest: +SKIP
+  ...         new_beam.axis2_array, new_beam.data_array[0, 0, 0, 0, :, 0], 'r')
+  >>> plt.xscale('log') # doctest: +SKIP
+  >>> plt.xlabel('Zenith Angle (deg)') # doctest: +SKIP
+  >>> plt.ylabel('Power') # doctest: +SKIP
+  >>> plt.show() # doctest: +SKIP
+
+
+  delete all written files
```

Comment:
This bit seems out of place. It gets rendered under "Selecting a range of Zenith Angles," which itself doesn't write any files. It looks like it's a catch-all for the whole tutorial, so maybe it should be in its own section.

Reply:
Yes, I'll make it better. It's actually there so that when you run the doctests it doesn't litter your directory with files...